cfme/cloud/instance/__init__.py

```diff
@@ -423,6 +423,65 @@ def power_control_from_cfme(self, *args, **kwargs):
         view.toolbar.power.item_select(kwargs.get('option'),
                                        handle_alert=not kwargs.get('cancel', False))
 
+    def set_ownership(self, user=None, group=None, click_cancel=False, click_reset=False):
+        """Set instance ownership
+
+        TODO: collapse this back to common.vm after both subclasses converted to widgetastic
+        Args:
+            user (str): username for ownership
+            group (str): groupname for ownership
+            click_cancel (bool): Whether to cancel form submission
+            click_reset (bool): Whether to reset form after filling
```

Comment:
@jkrocil I completely agree, and wanted to change this now - but it really should wait.

For now, this is callable both on infra.vm and cloud.instance in the same way, with the same arguments. I want to maintain this consistency until #5113 is merged and this method gets moved up into and overwrites cfme.common.vm.BaseVM.set_ownership - at which time I'll modify the arguments to remove 'click_'